api/cluster.go

```diff
@@ -382,15 +382,15 @@ func (s *Server) indexDelete(ctx *middleware.Context, req models.IndexDelete) {
 		return
 	}
 
-	defs, err := s.MetricIndex.Delete(req.OrgId, req.Query)
+	defs, err := s.metricsDeleteLocal(req.OrgId, req.Query)
```

Comment:
in many places in the code when a var is called `defs` it contains a slice of actual metric definitions (f.e. `[]idx.Archive`), so naming the count `defs` is a bit confusing, I'd name it like `defCount` or something like that.

Reply:
Updated call sites of `metricsDeleteLocal` w/ new names and added a comment to `metricsDeleteLocal` to indicate what the return type means.
ece7cffb7159214bc9fb62a6b2cadaeae0d576cb